### DISCUSSION TITLE
Don't use link color for autodoc names

### DISCUSSION THREAD

**Discussion by bhrutledge** [pradyunsg/furo#302] [Ideas] [answered]
As seen on https://pradyunsg.me/furo/kitchen-sink/autodoc/. I don't have a reference handy, but I think it's best to only use the link color for actionable UI elements.

This seems to be the source:

https://github.com/pradyunsg/furo/blob/edfbf30d8a9a1fdbff7553aca8036a389e88fbf4/src/furo/assets/styles/variables/_colors.scss#L36-L37

Where `--color-brand-content` is blue, which is also used for `--color-link`.

I don't have an alternate recommendation, but I might start with `--color-foreground-primary`.

**Comment by pradyunsg** [comment]
Links in Furo are not stylised as just a colour, but also have an underline visible under them.

I'm not too keen on tweaking the colours of the API presentation, but I'd be open to suggestions on how to improve contrast ratios of the link underline for API documentation lines.

**Reply by bhrutledge** [reply]
Yeah, I didn't see the underline in the links on https://pradyunsg.me/furo/kitchen-sink/autodoc/.

I opened https://github.com/pradyunsg/furo/pull/303 which uses the foreground color for API names; not sure where to continue the discussion.

Here's one reference re: link colors: https://www.nngroup.com/articles/guidelines-for-visualizing-links/. The relevant bit:

> Never show text in your chosen link colors unless it's a link.
> 
> You should generally avoid color for text unless it's a link. However, assuming it differs from the link color, you can sometimes use colored text without causing major usability problems. For example, in a checklist summary, you could show the word "okay" in green and the word "error" in red. (The fact that the word meanings are clearly different provides the required redundant cue for color-blind users.)
> 
> Don't use blue for non-link text, even if you don't use blue as your link color. Blue is still the color with the strongest perceived affordance of clickability.

**Reply by bhrutledge** [reply]
FWIW, I think of myself as a savvy web user, without disabilities, and I find the blue API names confusing. I feel an urge to click on them, and find them distracting from the clickable elements.

**Reply by pradyunsg** [reply]
Hmm... fair enough. Let's move the discussion over to the PR.

**Comment by pradyunsg** [comment, marked as answer]
Let's explore this in #303.
